src/WebJobs.Script.WebHost/Configuration/HttpOptionsSetup.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Configuration
+{
+    public class HttpOptionsSetup : IConfigureOptions<HttpOptions>
+    {
+        internal const int DefaultMaxConcurrentRequests = 100;
```

Comment:
I think the number of queuable requests should be fairly low. If a small instance can generally process 100 req/sec for a no-op function, then this current setting allows for 10 seconds of queued requests before we start returning 429s. I think that in practice this will mean that the throttle never really kicks in, because the scale controller logic of checking response latency will kick in first and scale the app out.

Lets spend some time tomorrow defining and running some tests to help us come up with the right default.

Reply:
Lets change the default outstanding to be double the default max concurrent, which would be 200.